cmd/medius/common/registry.go

```diff
@@ -100,7 +94,7 @@ func gatherArtifacts(registry *[]Entry, gatherers []api.ArtifactsGatherer) {
 		} else {
 			for i := range artifacts {
 				*registry = append(*registry, Entry{
-					Artifact:     artifacts[i],
+					Artifacts:    artifacts,
```

Comment:
The fedora gatherer needs an update to gather multiple architectures.

Reply:
Yes, right, I saw that it's not part of static `staticRegistry`.